core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1800,6 +1800,15 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  lazy val isInInterpreter: Boolean = {
```

Comment:
Why push this into `Utils`? It's not used elsewhere right? This class is turning into a generic uber class and I'd rather void that.